tf2/include/tf2/buffer_core.h

```diff
@@ -444,6 +444,6 @@ class BufferCore
 
 
 
-};
+}
```

Comment:
This is present in the upstream geometry2 code.  Would you mind opening a PR there to remove it (and then we'll inherit it the next time we merge)?  If you don't have time, let me know and I can do it instead.